docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -413,6 +413,12 @@ The following migration example allows users with the `Editor` role to access th
                     values: []
 ```
 
+## Update Solr configuration
+
+Solr configuration changes with the addition of new features such as spellchecking.
+
+[Generate the new configuration files using `generate-solr-config.sh`](install_solr.md#continue-solr-configuration).
```

Comment:
Solr configuration might be customized by 3rd-party, so we need to describe actual changes. As developer I would expect at least the same level of detail as provided in PR description: https://github.com/ibexa/solr/pull/52.

Reply:
e93fb0cf19581dbc9c84852aa86eccd078f9ef4d: I now expose the XML of the component and its assignation to /select; I moved the `generate-solr-config.sh` trick into a note box.